### DISCUSSION TITLE
Why is the Countdown '--value' property limited to a value <99 instead of 365?

### DISCUSSION THREAD

**Discussion by SeanM88** [saadeghi/daisyui#815] [Q&A] [answered]
Not sure if there is an easy work around I'm blanking on but it seems weird to have a Countdown timer demonstrating several countdowns using a "Days" tally but not allow a countdown to any date/time longer than 99 days.  Just wondering if there's someway to use a value larger than 99 for the Days element?

**Comment by saadeghi** [comment, marked as answer]
countdown is just the animation effect using CSS. it's not about days. it can be any number.  
By default, it's 99 at max but you can change it like this: https://play.tailwindcss.com/Uf9Hczurr6?file=css

**Reply by roymina** [reply]
but all numbers are 3 bit now

**Reply by saadeghi** [reply]
Yes. That's the point 🤔
